src/styles/bloks/cards.scss

```diff
@@ -0,0 +1,124 @@
+.cards {
+  --cards-count: 1;
+
+  display: grid;
+  grid-template-columns: repeat(var(--cards-count), 200px);
+  gap: 46px 48px;
+  justify-content: center;
+
+  padding: 50px 40px;
+
+  @media (min-width: 488px) {
+    & {
+      --cards-count: 2;
+    }
+  }
+
+  @media (min-width: 768px) {
+    & {
+      --cards-count: 3;
+    }
+  }
+
+  @media (min-width: 1024px) {
+    & {
+      --cards-count: 4;
+    }
+  }
+}
+
+.card {
+  box-sizing: border-box;
+  color: $color-nav;
+  width: 200px;
+  padding: 32px 16px 16px;
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  border: 1px solid #f3f3f3;
+
+  @include hover(transform, scale(1.2));
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &:hover {
+    & .card__title {
+      color: #34568b;
+    }
+  }
```

Comment:
The hover effect is a nice touch, but remember that it's not always necessary or desirable. In this case, the color change on hover might not be noticeable enough for users. Consider making the color change more significant or adding another visual cue to indicate that the element is interactive. This is also a recommendation.